www/js/app.js

```diff
@@ -1390,9 +1393,9 @@ define(['jquery', 'zimArchiveLoader', 'uiUtil', 'settingsStore','abstractFilesys
             document.getElementById('searchingArticles').style.display = 'none';
             // We have to remove the focus from the search field,
             // so that the keyboard does not stay above the message
-            $("#searchArticles").focus();
+            document.getElementById('searchArticles').focus();
             uiUtil.systemAlert("Archive not set : please select an archive", "No archive selected").then(function () {
```

Comment:
For the sake of consistency, change the double quotation marks in this systemAlert to single. I realize this wasn't part of the PR, but in this case, there's an opportunity to regularize the code in this area.